docs/source/io_formats/statepoint.rst

```diff
@@ -80,8 +80,10 @@ The current version of the statepoint file format is 17.0.
              dimension.
            - **Unstructured Mesh Only:**
               - **volumes** (*double[]*) -- Volume of each mesh cell.
-              - **centroids** (*double[]*) -- Location of the mesh cell
-                centroids.
+              - **vertices** (*double[]*) -- x, y, z values of the mesh vertices.
```

Comment:
It looks like we're missing: `type`, `filename`, `library`, `length_multiplier`

Reply:
Ah, I didn't see those prexisting missing data sets. Thanks!